aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -1020,8 +1021,9 @@ def _dump_schema(
             dbname = None
 
         self.log.info("Dumping schema from database %r", dbname)
+
         pg_dump_cmd = [
-            str(self.pgbin / "pg_dump"),
+            f"{self.pgbin}/pg_dump",
```

Comment:
I don't think this change is a win, the path construction should probably be left to Path instead of doing it by string formatting.

Reply:
@kmichel-aiven 
Thanks for your feed back.  Based on your suggestion does this look better?
```
Path(self.pgbin,"pg_dump")
```
If I am not mistaken the current code is trying to divide two strings... probably have never worked?

Reply:
The syntax is surprising at first but it's not dividing two strings, it's a syntax with `Path` objects that was added near python 3.4:
```python
>>> p = Path('/etc')
>>> q = p / 'init.d' / 'reboot'
>>> q
PosixPath('/etc/init.d/reboot')
```
So the existing code was working.

Reply:
Thank you @kmichel-aiven 
Learned something new and should have noticed `self.pgbin` is a `Path`!
I reverted these changes.